tests/factories.py

```diff
@@ -15,20 +15,12 @@ class Meta:
 class SubstanceFactory(factory.Factory):
 
     id = factory.Sequence(lambda n: f"DTXCID{n:09}")
-    identifiers = {
-        "preferred_name": "Moperone",
-        "display_name": "Moperone",
-        "casrn": "1050-79-9",
-        "inchikey": "AGAHNABIDCTLHW-UHFFFAOYSA-N",
-        "casalts": [
-            {"casalt": "0001050799", "weight": 0.5},
-            {"casalt": "1050799", "weight": 0.5},
-        ],
-        "synonyms": [
-            {"synonym": "Meperon", "weight": 0.75},
-            {"synonym": "Methylperidol", "weight": 0.5},
-        ],
-    }
+    identifiers = (
+        '{ "preferred_name":"Moperone","display_name":"Moperone","casrn":"1050-79-9",'
+        '"inchikey": "AGAHNABIDCTLHW-UHFFFAOYSA-N", '
+        '"casalts":[{"casalt":"0001050799","weight:0.5},{"casalt":"1050799","weight":0.5}],'
```

Comment:
there is a missing closing `"` here on weight, and I can't get tests to pass unless I remove the single quotes on either side `'` which was the same issue we were having with loading seed data when doing 142.
```suggestion
        '"casalts":[{"casalt":"0001050799","weight":0.5},{"casalt":"1050799","weight":0.5}],'
```